config/routes.rb

```diff
@@ -24,7 +24,10 @@
   get "forgot_password_confirmation", to: "forgot_passwords#confirm"
   resources :password_resets, only: [:show, :create]
   resources :forgot_passwords, only: [:create]
-  get "expired_token", to: "password_resets#expired_token"
+  get "expired_token", to: "pages#expired_token"
+
+  resources :invitations, only: [:new, :create]
+  get 'register/:token', to: "users#new_with_invitation_token", as: "register_with_token"
```

Comment:
Line is too long. [89/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.